src/modules/ttlmonitor/generators/ReadoutGeneratorVMM.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 European Spallation Source, ERIC. See LICENSE file
```

Comment:
Should this live somewhere different, maybe the VMM folder? And it isn't based on LoKI, the description is a bit off

Reply:
I'll update the comment. No need to move this. It will be removed as soon as we have reached an agreement 
that we're using the ttlmon ICD.